blaseball_mike/models/league.py

```diff
@@ -1,35 +1,34 @@
 from collections import OrderedDict
 
-from .base import Base
+from .base import Base, BaseChronicler
 from .team import Team
-from .. import database
 
 
-class League(Base):
+class League(BaseChronicler):
+    _entity_type = "league"
+    ILB = 'd8545021-e9fc-48a3-af74-48685950a183'
+
     """
     Represents the entire league
     """
     @classmethod
     def _get_fields(cls):
-        p = cls.load()
+        p = cls.load(cls.ILB)
         return [cls._from_api_conversion(x) for x in p.fields]
 
     def __init__(self, data):
         super().__init__(data)
         self._teams = {}
 
     @classmethod
-    def load(cls):
-        return cls(database.get_league())
-
-    @classmethod
-    def load_by_id(cls, id_):
-        return cls(database.get_league(id_))
+    def load_ilb(cls, id_=ILB, *args, **kwargs):
+        return cls.load_one(id_, *args, **kwargs)
 
     @Base.lazy_load("_subleague_ids", cache_name="_subleagues", default_value=dict())
     def subleagues(self):
         """Returns dictionary keyed by subleague ID."""
-        return {id_: Subleague.load(id_) for id_ in self._subleague_ids}
+        timestamp = self.timestamp if hasattr(self, "timestamp") else None
+        return {id_: Subleague.load_one(id_, time=timestamp) for id_ in self._subleague_ids}
```

Comment:
```suggestion
        return {id_: Subleague.load_one(id_, time=getattr(self, "timestamp", None)) for id_ in self._subleague_ids}
```

Reply:
We should also probably have this return an array of subleagues rather than a dictionary, since that's what the blaseball API defines should be here